pycodestyle.py

```diff
@@ -124,7 +124,7 @@
 STARTSWITH_DEF_REGEX = re.compile(r'^(async\s+def|def)')
```

Comment:
You will also have to add `\b` here, as lines starting with `async? def` are special-cased in `def compound_statements(logical_line):`